feature: define Staged Configs (#264)
nginx/documentation: content/nginx-one/how-to/nginx-configs/view-edit-nginx-configurations.md

---
# We use sentence case and present imperative tone
title: View and edit NGINX configurations
# Weights are assigned in increments of 100: determines sorting order
weight: 300
# Creates a table of contents and sidebar, useful for large documents
toc: true
# Types have a 1:1 relationship with Hugo archetypes, so you shouldn't need to change this
type: tutorial
# Intended for internal catalogue and search, case sensitive:
product: NGINX One
---
<!-- Possible future include, with similar files in config-sync-groups/ and staged-configs/ subdirectories -->

## Overview

This guide explains how to add a **Instances** to your NGINX One Console. 

## Before you start

Before you add **Instances** to NGINX One Console, ensure:

> - You have administrator access to NGINX One Console.

Author: ```suggestion
- You have an NGINX One Console account with staged configuration permissions.```

---
# We use sentence case and present imperative tone
title: View and edit NGINX configurations
# Weights are assigned in increments of 100: determines sorting order
weight: 300
# Creates a table of contents and sidebar, useful for large documents
toc: true
# Types have a 1:1 relationship with Hugo archetypes, so you shouldn't need to change this
type: tutorial
# Intended for internal catalogue and search, case sensitive:
product: NGINX One
# Intended for internal catalogue

> docs: "DOCS-000"

Reviewer: Leaving this out completely serves the same purpose as adding DOCS-000, and is preferred.

Reviewer: ```suggestion

```

> # Intended for internal catalogue
> docs: "DOCS-000"

Author: ```suggestion
```
